scripts/sst_prep_obs.sh

```diff
@@ -2,15 +2,13 @@
 
 while getopts "i:d:" opt; do
    case $opt in
-      i) sst_source=("$OPTARG");;
-      d) filename=("$OPTARG");;
+      i) SSTsource=("$OPTARG");;
```

Comment:
Not sure why you would suddenly change variable name ... sst_source is fine.

Reply:
As mentioned in the reply to the previous comment.